roles/etcd_common/README.md

```diff
@@ -1,17 +1,27 @@
 etcd_common
 ========================
 
-TODO
+Common resources for dependent etcd roles. E.g. default variables for:
+* config directories
+* certificates
+* ports
+* other settings
 
-Requirements
-------------
+Or `delegated_serial_command` ansible module for executing a command on a remote node. E.g.
 
-TODO
+```yaml
+- delegated_serial_command:
+    command: /usr/bin/make_database.sh arg1 arg2
+    creates: /path/to/database
+```
 
-Role Variables
---------------
+Or etcdctl.yml playbook for installation of `etcdctl` aliases on a node. E.g.
 
-TODO
+```yaml
+- include_role:
+    name: etcd_common
+    tasks_from: etcdctl
+```
```

Comment:
Get rid of this in favor of the example playbook below? Just seems like duplication.